pkg/helpers/special_values.go

```diff
@@ -0,0 +1,60 @@
+package helpers
```

Comment:
We should move away from this package (use internal/*); we will do with a bigger cleanup but as the general rule, but if we are moving them now, let's place them more correctly, e.g. we have:
- internal/provider
- internal/provider/docs
maybe we could add internal/provider/validators for validations (then usage is nicer - validators.ValidateXyz and we won't have to move it again), special values may reside it  internal/provider and helpers/doc can go to internal/provider/docs.

Reply:
Done.